tests/unit/AddEditOrgPerson.spec.ts

```diff
@@ -7,6 +7,9 @@ import { getVuexStore } from '@/store'
 import AddEditOrgPerson from '@/components/common/AddEditOrgPerson.vue'
 import { EmptyOrgPerson } from '@/interfaces'
 
+// mock the console.warn function to hide "[Vuetify] Unable to locate target XXX"
+console.warn = jest.fn()
```

Comment:
The warning is because the shared ConfirmDialog expected to attach itself to its parent component, but for some reason Jest isn't setting the parent's ID, so the dialog can't find its attachment.

The "attach" prop was no present in the previous, local version of ConfirmDialog, which avoided this issue but unfortunately did not attach the dialog to the desired parent. (Didn't seem to matter in practise but still...)